app/views/layouts/_navbar.html.erb

```diff
@@ -25,20 +25,19 @@
       </ul>
       <ul class="nav navbar-nav navbar-right">
         <li><a href="/cart"><span class='glyphicon glyphicon-shopping-cart'></span> View Cart</a></li>
-        <% if current_user %>
-          <li><a href="#"> <%= "Logged in as #{current_user.username}" %> </a></li>
-          <li> <%= link_to "Logout", login_path, method: :delete %></li>
-        <% else %>
-          <li><a href="/login">Login</a></li>
-        <% end  %>
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            <li><a href="#">Action</a></li>
-            <li><a href="#">Another action</a></li>
-            <li><a href="#">Something else here</a></li>
-            <li role="separator" class="divider"></li>
-            <li><a href="#">Separated link</a></li>
+        <li class="dropdown account-dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Account <span class="caret"></span></a>
+          <ul class="dropdown-menu acct-dropdown-menu">
+            <% if current_user %>
+              <% if current_admin %>
+                <li><a href="<%=  admin_user_path(current_user.id)  %>">Admin Dashboard</a></li>
+              <% end %>
+            <li><a href="#"> <%= "Logged in as #{current_user.username}" %> </a></li>
+            <li> <%= link_to "Logout", login_path, method: :delete %></li>
+          <% else %>
+            <li><a href="/users/new">Signup</a></li>
+            <li><a href="/login">Login</a></li>
+          <% end  %>
```

Comment:
Here is the drop down that is in there.  I can move this back around to the normal if needed